<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CuteMarkEd</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="MarkdownEditor" name="plainTextEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
      <widget class="QWebView" name="webView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="htmlSourceTextEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExportToHTML"/>
    <addaction name="actionExportToPDF"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuFormat">
     <property name="title">
      <string>Format</string>
     </property>
     <addaction name="actionStrong"/>
     <addaction name="actionEmphasize"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCopyHtmlToClipboard"/>
    <addaction name="separator"/>
    <addaction name="menuFormat"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuStyles">
     <property name="title">
      <string>Styles</string>
     </property>
     <addaction name="actionDefault"/>
     <addaction name="actionGithub"/>
     <addaction name="actionSolarizedLight"/>
     <addaction name="actionSolarizedDark"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuStyles"/>
    <addaction name="separator"/>
    <addaction name="actionSplit_1_1"/>
    <addaction name="actionSplit_2_1"/>
    <addaction name="actionSplit_1_2"/>
    <addaction name="actionSplit_3_1"/>
    <addaction name="actionSplit_1_3"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Table of contents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWebView" name="tocWebView">
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionGithub">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Github</string>
   </property>
  </action>
  <action name="actionSolarizedLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Solarized Light</string>
   </property>
  </action>
  <action name="actionSolarizedDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Solarized Dark</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About CuteMarkEd...</string>
   </property>
  </action>
  <action name="actionExportToHTML">
   <property name="text">
    <string>Export to HTML...</string>
   </property>
  </action>
  <action name="actionExportToPDF">
   <property name="text">
    <string>Export to PDF...</string>
   </property>
  </action>
  <action name="actionSplit_1_1">
   <property name="text">
    <string>Split 1:1</string>
   </property>
  </action>
  <action name="actionSplit_2_1">
   <property name="text">
    <string>Split 2:1</string>
   </property>
  </action>
  <action name="actionSplit_1_2">
   <property name="text">
    <string>Split 1:2</string>
   </property>
  </action>
  <action name="actionSplit_3_1">
   <property name="text">
    <string>Split 3:1</string>
   </property>
  </action>
  <action name="actionSplit_1_3">
   <property name="text">
    <string>Split 1:3</string>
   </property>
  </action>
  <action name="actionCopyHtmlToClipboard">
   <property name="text">
    <string>Copy HTML to Clipboard</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionStrong">
   <property name="text">
    <string>Strong</string>
   </property>
  </action>
  <action name="actionEmphasize">
   <property name="text">
    <string>Emphasize</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>MarkdownEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>markdowneditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>plainTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUndo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRedo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editRedo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDefault</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>styleDefault()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGithub</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>styleGithub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSolarizedLight</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>styleSolarizedLight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSolarizedDark</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>styleSolarizedDark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>helpAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportToHTML</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileExportToHtml()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportToPDF</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileExportToPdf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tocWebView</sender>
   <signal>linkClicked(QUrl)</signal>
   <receiver>MainWindow</receiver>
   <slot>tocLinkClicked(QUrl)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSplit_1_1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewChangeSplit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSplit_2_1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewChangeSplit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSplit_1_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewChangeSplit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSplit_3_1</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewChangeSplit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSplit_1_3</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>viewChangeSplit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitter</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopyHtmlToClipboard</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editCopyHtml()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCut</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCopy</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPaste</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStrong</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editStrong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEmphasize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editEmphasize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Print</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>filePrint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
